pkg/config/template.go

```diff
@@ -1219,6 +1219,8 @@ const templateStringCrioRuntimeRuntimesRuntimeHandler = `# The "crio.runtime.run
 # allowed_annotations = []
 # platform_runtime_paths = { "os/arch" = "/path/to/binary" }
 # no_sync_log = false
+# inherit_default_runtime = []
```

Comment:
Are we missing this one? I thought we've added this as part of other change. Also, this is a boolean, I think.

Reply:
It's already available in [L1213](https://github.com/cri-o/cri-o/pull/8829/files#diff-22a2b88ec18d80ceabdae558506a636efbaf4d9008ba36bfe93db19d8f0be113R1213)

Reply:
oopsies good catch! I didn't look hard enough for it :smile: